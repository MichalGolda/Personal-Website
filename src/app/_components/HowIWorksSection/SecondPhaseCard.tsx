import PhaseCard from "./PhaseCard";
import { PhaseCardColors } from "./constants";

const SecondPhaseCard = () => (
  <PhaseCard
    className="w-full md:w-1/2 md:ml-[calc(50%/3)]"
    headline="faza 2"
    title="Design UX/UI"
    color={PhaseCardColors.PHASE_SECOND}
    body="Na bazie zebranych informacji tworzę projekt graficzny. W trakcie projektowania regularnie konsultuję się z klientem, aby na bieżąco uwzględniać wprowadzane zmiany. Przechodzę do kolejnego etapu dopiero po uzyskaniu pełnej akceptacji."
    data-aos="fade-right"
    data-aos-duration="1000"
  />
);

export default SecondPhaseCard;
