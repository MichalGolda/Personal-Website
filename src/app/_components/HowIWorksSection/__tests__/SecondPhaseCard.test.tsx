import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import SecondPhaseCard from "../SecondPhaseCard";

describe("SecondPhaseCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<SecondPhaseCard />);

    expect(container).toMatchSnapshot();
  });
});
