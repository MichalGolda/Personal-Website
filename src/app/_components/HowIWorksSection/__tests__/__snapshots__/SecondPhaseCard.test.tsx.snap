// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondPhaseCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4 border border-solid border-lightGrey p-4 bg-white w-full md:w-1/2 md:ml-[calc(50%/3)]"
    data-aos="fade-right"
    data-aos-duration="1000"
  >
    <div
      class="flex flex-col"
    >
      <span
        class="before:block flex items-center gap-x-2 text-sm md:text-base font-semibold uppercase before:w-2 before:h-2 before:bg-[#FFB930] text-[#FFB930]"
      >
        faza 2
      </span>
      <h4
        class="text-lg md:text-xl lg:text-2xl text-secondary font-bold text-secondary"
      >
        Design UX/UI
      </h4>
    </div>
    <p
      class="text-body text-sm md:text-base"
    >
      Na bazie zebranych informacji tworzę projekt graficzny. W trakcie projektowania regularnie konsultuję się z klientem, aby na bieżąco uwzględniać wprowadzane zmiany. Przechodzę do kolejnego etapu dopiero po uzyskaniu pełnej akceptacji.
    </p>
  </div>
</div>
`;
